staker/staker.gno

```diff
@@ -43,45 +43,52 @@ func Init(token grc20.Token) {
 }
 
 // StakeAmount stakes the specified amount for the given delegate
-func StakeAmount(delegate std.Address, amount *u256.Uint) error {
+func StakeAmount(delegate string, amount uint64) error {
```

Comment:
Previous code was using u256 for amount type. Which means number can be exceed default uint64 type.

However, it's currently using uint64.

If uint64 is enough type, resolve this conversation.
If not, use string for amount then convert to u256.

Reply:
amount param type changed: uint64 -> string

https://github.com/gnoswap-labs/governance/commit/6fd92005e9110f1ba48fbcf35e2d2eb2bbd19aa0